x-pack/plugins/ingest_manager/server/services/epm/fields/field.ts

```diff
@@ -108,7 +108,15 @@ function dedupFields(fields: Fields): Fields {
       return f.name === field.name;
     });
     if (found) {
-      if (found.type === 'group' && field.type === 'group' && found.fields && field.fields) {
+      if (
```

Comment:
Would be nice to have a comment in here that explains why this code is needed (to cover the a, a.b problem).